core/controllers/acl_decorators.py

```diff
@@ -1657,49 +1655,6 @@ def test_can_delete_question(self, question_id, **kwargs):
     return test_can_delete_question
 
 
-def can_edit_subtopic_page(handler):
```

Comment:
Why delete? /cc @aks681 to comment on whether this is OK.

Reply:
I did not see this decorator used anywhere, so I deleted it

Reply:
OK, thanks for the explanation -- I want to get confirmation from @aks681 on this, just in case it's a mistake on the new structures end. @aks681 any thoughts?

Reply:
This is interesting. I checked the decorator being used for EditableSubtopicPageDataHandler, and it is `can_view_any_topic_editor`. So, this decorator is not being used anywhere. If this is being removed, then there is no use of the function `check_can_edit_subtopic_page` in  topic_services as well.
Subtopic page editor is indeed a subset of the topic editor, so I think `can_view_topic_editor` is fine as a check for this also?

Reply:
can_view_topic_editor sounds good to me for this, yep.

@lilithxxx could you please make the appropriate updates?

Reply:
I am sorry but I don't understand what I need to do. @aks681 you are right there is no use of `check_can_edit_subtopic_page` in topic_services. Should I remove that? I don't understand these two lines that you said:

> Subtopic page editor is indeed a subset of the topic editor, so I think `can_view_topic_editor` is fine as a check for this also?

I did not find any `can_view_topic_editor` decorator in our codebase. Do you mean `can_view_any_topic_editor`? And if so what should I do with this decorator?

Reply:
Go with can_view_any_topic_editor. Drop this decorator.

/cc @aks681 to confirm that this is correct.

Reply:
Yes but where should I use can_view_any_topic_editor? This decorator (can_edit_subtopic_page) is not used anywhere so there is nothing to replace

Reply:
No need to do anything with `can_view_any_topic_editor`. Just remove this and remove `check_can_edit_subtopic_page` from topic_services. That's it.

Reply:
Done